src/types/mapper.ts

```diff
@@ -24,6 +24,14 @@ export interface FILESYSTEM_MAPPER_INPUT {
   configuration: any;
 }
 
+export interface UPDATE_FILESYSTEM_MAPPER_INPUT {
```

Comment:
Change (Suggestion): we can change these interfaces to Pascal Case, only the queries and mutations should be using Uppercase and snake case